src/lib.rs

```diff
@@ -270,14 +305,24 @@ impl SyncEditorSystem {
     }
 }
 
-impl Default for SyncEditorSystem {
-    fn default() -> Self { SyncEditorSystem::new() }
-}
-
 impl<'a> System<'a> for SyncEditorSystem {
     type SystemData = Entities<'a>;
 
     fn run(&mut self, entities: Self::SystemData) {
+        // Determine if we should send full state data this frame.
+        let now = Instant::now();
+        let send_this_frame = now > self.next_send;
```

Comment:
Should probably be >=

Reply:
Haha, that's a good point. Gotta get that nanosecond precision, I guess 😜